modules/miscutil/lib/upgrades/invenio_2015_01_13_hide_holdings.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2014 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation,
+# Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Upgrade for explicitly hiding the 'holdings' tab for all collections\
+ except 'Books' without an existing rule"""
+
+from invenio.dbquery import run_sql
+
+depends_on = ['invenio_release_1_1_0']
+
+
+def info():
+    """Upgrade recipe information."""
+    return "Updates the collectiondetailedrecordpagetabs to hide 'holdings' for every collection \
+            except 'Books' without an existing rule."
+
+
+def do_upgrade():
```

Comment:
@blixhavn I see your point.  However, collection 1 usually does not have any records attached to it, so even if people do have settings for it, they do not see its results.  If they had set something up without clicking on "apply for subcollections", then why not to take advantage of this change and propose those instead?  But I agree with you that this is an ultra corner case (kind of equal to forgotten click on "apply for subcollections") and that it may be better to preserve Atlantis way anyway.  Depends on whether we look at this situation more as a bug or as a feature.  I guess I tended to prefer bug-point-of-view, while you opted for feature-point-of-view.    Since "apply for subcollections" is unchecked by default in the UI, you are probably right that it may be better to look at this as at a feature...